db/seeds/mock_offers.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Like I mentioned in the description, this is mostly just a guess. This isn't the official format.